src/commands/upload_file.ts

```diff
@@ -69,14 +86,31 @@ new CLICommand({
 						driveKey
 					};
 				});
+
 				return fileParameters;
 			}
 
+			const parentFolderId: FolderID = parameters.getRequiredParameterValue(ParentFolderIdParameter, EID);
+			const globValue = parameters.getParameterValue(GlobParameter);
+
+			if (globValue) {
+				const files = glob.sync(globValue);
```

Comment:
Is there a reason why we don't use the standard shell glob functionality?

We could get rid of this dep if the only stuff it is doing is resolving the glob into an array of paths. That's what the shell currently does

Reply:
I'll clone this stuff and hack it locally to see if that's possible

Reply:
We could. Each shell can have different rules for glob and using node-glob just create a standard.
I think it's just a different approach, I don't know if we want a standard way of glob or not.
Another thing is that I'm not sure if all shells will return a list of paths.

Reply:
For example, my current zsh configuration will only return the first path.

Reply:
Ok, found something :D
If we change the parameter type to array shell glob will work and we don't need node-glob.
So every shell can have different glob rules, and users will need to know how to use them. It's the same for node-glob, the difference is that we have a standard to give the user if they need.
I also saw that every shell glob should return a list of paths, so I think we're good if we wanto to go with shell glob instead of node-glob. I tried with default glob rules of zsh, bash, dash and fish, which I think are the most popular ones.

<img width="523" alt="image" src="https://user-images.githubusercontent.com/9200155/145279424-99e170e9-ee06-4896-ad1a-8b3e67071bff.png">

Reply:
See here: https://github.com/karlprieb/ardrive-cli/pull/1

Reply:
The latest changes are moved to https://github.com/ardriveapp/ardrive-cli/pull/185 for convenience